### DISCUSSION TITLE
'PolyData' object has no attribute 'reconstruct_surface'

### DISCUSSION THREAD

**Discussion by adam-grant-hendry** [pyvista/pyvista#2633] [Q&A]
When running this code

[Docs Surface Reconstruction Example](https://dev.pyvista.org/examples/01-filter/surface_reconstruction.html#surface-reconstruction-example)

I get the error

`'PolyData' object has no attribute 'reconstruct_surface'`

**Comment by akaszynski** [comment]
This will be out in 0.32.0, I'm almost ready to create the release PR.

**Comment by adamgranthendry** [comment]
Ah, I'm sorry. False positive. I'll close this issue then. Thanks!

**Comment by adam-grant-hendry** [comment]
Closing as this is a WIP.
